subt/zmq-subt-x4.json

```diff
@@ -86,6 +86,14 @@
 	      "init": {
 	        "outputs": ["rot", "acc", "orientation", "top_scan", "bottom_scan", "pose3d"]
 	      }
+      },
+      "lora": {
+          "driver": "lora",
+          "in": ["raw", "pose2d"],
```

Comment:
You know this is not needed here, right? And in this PR `lora` also accepts `radio`...